app/views/buildings/new.html.erb

```diff
@@ -6,7 +6,7 @@
     <%= f.input :address, label: "Address" %>
     <%= f.input :mortgage, label: "Mortgage" %>
     <%= f.input :purchase_price, label: "Purchase price" %>
-    <%= f.input :purchase_date, label: "Purchase date" %>
+    <%= f.input :date, as: :date, label: "Purcahse Date:", start_year: Date.today.year, order: [:day, :month, :year], html5: true, input_html: { min: Date.today.to_s, value: Date.today.to_s } %>
```

Comment:
I think, we need to keep the f.input :purchase_date because it's the column we have in our schema.
Maybe as: date too --> to be confirmed on Google.
Label: "Purchase Date:" instead of "Purcahse Date:"